File: src/main/kotlin/com/stannis/dataModel/statementTypes/If.kt
package com.stannis.dataModel.statementTypes

import com.stannis.dataModel.Statement

data class If(
    override val `$type`: String? = "If",
    var condition: Statement?,
    var thenClause: Statement?,
    var elseClause: Statement?,
    var condDecl: Statement?
) : Statement

File: Interpreter/Interpreter/Models/serialize/statementTypes/If.cs
namespace Interpreter.Models.serialize.statementTypes{

    public class If : IStatement
    {
        public IStatement condition { set; get; }
        
        public IStatement thenClause { set; get; }

        public IStatement elseClause { set; get; }

        public IStatement condDecl { set; get; }
        
    }

};